WeakAuras/Prototypes.lua

```diff
@@ -8036,6 +8038,58 @@ Private.event_prototypes = {
           limit = 2
         },
       },
+      {
+        name = "name",
+        display = L["Unit Name"],
+        type = "string",
+        store = true,
+        hidden = true,
+        test = "true"
+      },
+      {
+        name = "realm",
```

Comment:
Can this ever have a value? Since you can only have threat on npcs

Reply:
Actually due to how UnitNameWithRelam works, it probably always the realm of the player.